site/content/docs/5.1/components/modal.md

```diff
@@ -644,7 +644,7 @@ Our default modal without modifier class constitutes the "medium" size modal.
   <div class="modal-dialog modal-xl">
     <div class="modal-content">
       <div class="modal-header">
-        <h5 class="modal-title h4" id="exampleModalXlLabel">Extra large modal</h5>
```

Comment:
Wouldn't it be possible to keep the `.h4` class which [has already the right values for desktop and tablet](https://boosted.netlify.app/docs/5.1/content/typography/#headings) and just add a media breakpoint rule to only fix the mobile value (with a selector that targets `.h4`)?